dusty/cli/__init__.py

```diff
@@ -125,7 +126,10 @@ def main():
     args = docopt(__doc__, options_first=True)
     command, command_args = args['<command>'], args['<args>']
     if command is None:
-        print __doc__.strip()
+        if '-v' in sys.argv:
+            print 'Dusty version {}'.format(constants.VERSION)
```

Comment:
@thieman i do not remember, just want to make sure. -v is for the client, version is for the deamon (that would make sense)